crates/bevy_ui/src/widget/image.rs

```diff
@@ -1,29 +1,16 @@
 use crate::{CalculatedSize, Size, UiImage, Val};
 use bevy_asset::Assets;
 use bevy_ecs::{
-    component::Component,
-    query::{With, Without},
-    reflect::ReflectComponent,
+    query::Without,
     system::{Query, Res},
 };
-use bevy_reflect::{Reflect, ReflectDeserialize, ReflectSerialize};
 use bevy_render::texture::Image;
 use bevy_text::Text;
-use serde::{Deserialize, Serialize};
-
-/// Describes how to resize the Image node
-#[derive(Component, Debug, Default, Clone, Reflect, Serialize, Deserialize)]
-#[reflect(Component, Serialize, Deserialize)]
-pub enum ImageMode {
-    /// Keep the aspect ratio of the image
-    #[default]
-    KeepAspect,
-}
 
 /// Updates calculated size of the node based on the image provided
 pub fn image_node_system(
```

Comment:
We should rename this system too.

Reply:
`update_image_calculated_size_system`?

Are we moving away from `*_system` naming, usage seems inconsistent atm?